pkgs/tools/networking/vpnc-scripts/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchgit
+, makeWrapper
+, nettools, gawk, openresolv, coreutils, gnugrep
+}:
+
+let
+  os = if stdenv.isDarwin then "Darwin" else "Linux";
+
```

Comment:
```suggestion
vpncBinPath = [ nettools gawk coreutils gnugrep ] ++ lib.optional stdenv.isLinux [ openresolv ];
```

Reply:
I'd save the conditional binPath in a variable to use it afterwards.

Reply:
We don't need to put something on a variable if we only use it once.